src/views/Stake/StakeRow.tsx

```diff
@@ -0,0 +1,30 @@
+import { Typography } from "@material-ui/core";
+import { Skeleton } from "@material-ui/lab";
+import { Trans } from "@lingui/macro";
+
+interface StakeRowProps {
+  title: string;
+  indented?: boolean;
+  id?: string;
+  balance: string;
+  isAppLoading?: boolean;
+}
+
+const StakeRow = (props: StakeRowProps) => {
+  return (
+    <div className="data-row" style={props.indented ? { paddingLeft: "10px" } : {}}>
+      <Typography variant={props.indented ? `body2` : `body1`} color={props.indented ? "textSecondary" : "primary"}>
+        <Trans>{props.title}</Trans>
```

Comment:
I'm getting a parsing error I think on this line. see below.
I think this `<Trans>` needs to be moved upstream to where the prop is passed.

![image](https://user-images.githubusercontent.com/80423742/144656257-0ec0a178-be45-41bd-8cec-7a918ada53d6.png)

Reply:
hmmm. I'm not seeing this from my end, but I see now that strings are only appearing in english. nonetheless I think you're right in lifting the translation up. 
I'll need to dig in more as to why I don't see what you're seeing and why i'm falling back to english strings locally. At first glance i'm seeing a: 
`I18nProvider did not render. A call to i18n.activate still needs to happen or forceRenderOnLocaleChange must be set to false.` message in the console.

I'll get this change pushed up shortly.